### PR TITLE
Remove RHEL6 content

### PR DESCRIPTION
**Pull request by redhatrises**
- RHEL6 is no longer under general support

### CODE REVIEW

**Review by JAORMX — APPROVED** [review]

File: linux_os/guide/system/software/integrity/endpoint_security_software/configure_user_data_backups/rule.yml [hunk 1]
```diff
@@ -12,18 +12,10 @@ rationale: |-
     Operating system backup is a critical step in maintaining data assurance and
     availability. User-level information is data generated by information system
     and/or application users. Backups shall be consistent with organizational
-    recovery time and recovery point objectives.false
```
Comment (jan-cerny) [inline linux_os/guide/system/software/integrity/endpoint_security_software/configure_user_data_backups/rule.yml:15]: :+1:

File: linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/bash/shared.sh [hunk 2]
```diff
@@ -1,11 +1,7 @@
 # platform = multi_platform_wrlinux,multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
 
-# Find which files have incorrect hash {{{ "(not in /etc, because there are all system related config. files) " if product == "rhel6" }}}and then get files names
-{{%- if product == "rhel6" %}}
-files_with_incorrect_hash="$(rpm -Va | grep -E '^..5.* /(bin|sbin|lib|lib64|usr)/' | awk '{print $NF}' )"
-{{%- else %}}
+# Find which files have incorrect hash {{{ "(not in /etc, because there are all system related config. files) " }}} and then get files names
```
Comment (jan-cerny) [inline linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/bash/shared.sh:3]: this looks like an unwanted Jinja invocation, there was probably some collision in past

File: rhel7/kickstart/ssg-rhel7-cis-ks.cfg [hunk 3]
```diff
@@ -59,7 +59,7 @@ network --onboot yes --device eth0 --bootproto dhcp --noipv6
 # Plaintext password is: server
 # Refer to e.g. https://pykickstart.readthedocs.io/en/latest/commands.html#rootpw to see how to create
 # encrypted password form for different plaintext password
-rootpw --iscrypted $6$rhel6usgcb$aS6oPGXcPKp3OtFArSrhRwu6sN8q2.yEGY7AIwDOQd23YCtiz9c5mXbid1BzX9bmXTEZi.hCzTEXFosVBI5ng0
+rootpw --iscrypted $6$rhel7usgcb$aS6oPGXcPKp3OtFArSrhRwu6sN8q2.yEGY7AIwDOQd23YCtiz9c5mXbid1BzX9bmXTEZi.hCzTEXFosVBI5ng0
```
Comment (jan-cerny) [inline rhel7/kickstart/ssg-rhel7-cis-ks.cfg:62]: ?

Comment (jan-cerny) [reply rhel7/kickstart/ssg-rhel7-cis-ks.cfg:62]: It's also in multiple other kickstarts.

File: shared/templates/template_BASH_accounts_password [hunk 4]
```diff
@@ -6,17 +6,10 @@
 . /usr/share/scap-security-guide/remediation_functions
 {{{ bash_instantiate_variables("var_password_pam_" + VARIABLE) }}}
 
-{{% if product == "rhel6" %}}
-{{# There is no package libpwquality for RHEL6 #}}
-
```
Comment (jan-cerny) [inline shared/templates/template_BASH_accounts_password:11]: The code from here until the `{{% else %}}` needs to be removed as well.

File: tests/unit/ssg-module/test_rule_dir_stats.py [hunk 5]
```diff
@@ -42,85 +42,10 @@ def test_missing_remediation():
...
-        "id": "bad_rule",
-        "ovals": {}
-    }
-
-    assert rds.two_plus_oval(three_rule)
-    assert rds.two_plus_oval(two_rule)
-    assert not rds.two_plus_oval(one_rule)
-    assert not rds.two_plus_oval(empty_rule)
-
-
```
Comment (jan-cerny) [inline tests/unit/ssg-module/test_rule_dir_stats.py:77]: I think it shouldn't be removed, it seems to be harmless unit test.

**Review by jan-cerny — COMMENTED** [review]
Great!

File: rhel7/kickstart/ssg-rhel7-cis-ks.cfg (hunk 3 above)
Comment (redhatrises) [reply rhel7/kickstart/ssg-rhel7-cis-ks.cfg]: Unfortunately, yes

Comment (ggbecker) [reply rhel7/kickstart/ssg-rhel7-cis-ks.cfg]: this parameter seems to be the `salt` used in the password generation. I think that changing this string might break the password setting

https://pykickstart.readthedocs.io/en/latest/commands.html#rootpw

Comment (ggbecker) [reply rhel7/kickstart/ssg-rhel7-cis-ks.cfg]: Ok, I've seen you removed from the pull request.

We can later change to a generic `salt`, for example:

`python -c 'import crypt; print(crypt.crypt("server", "$6$s4lt"))'`
`$6$s4lt$yASNkSe7qTgJkwSG8EohHKOKF4tyANeZhiIf02D9/U5XzUtsGMOFI7fYonLGcXN.zEOdt/rLpuLTm8Ile8.K21`

**Pull request merged**